src/firefly/js/templates/common/ttFeatureWatchers.js

```diff
@@ -174,9 +182,12 @@ export const PrepareDownload = React.memo(({table_id, tbl_title, viewerId, showF
         centerColValues: { ra, dec }
     };
 
+
+    const isRowSelected = selectInfoCls.getSelectedCount()>0;
+
     return (
         <>
-            {loading && <div />}
+            {loading && <ToolbarButton enabled={false} variant={isRowSelected?'solid':'soft'} color='warning' text={downloadType === 'script' ? 'Generate Download Script' : 'Prepare Download'}/>}
```

Comment:
came up with a solution to show a dummy (disabled) button while the `semList` is loading so as to not have any weird UX movement of a button randomly appearing for the user @robyww.